pkgs/applications/misc/monitorcontrol/default.nix

```diff
@@ -1,19 +1,40 @@
-{ lib, fetchurl, stdenv, undmg }:
+{ lib, fetchurl, stdenv, unzip }:
 
 # This cannot be built from source due to the problematic nature of XCode - so
 # this is what it's like when doves cry?
 
 stdenv.mkDerivation rec {
   pname = "MonitorControl";
-  version = "4.1.0";
+  version = "4.2.0";
 
   src = fetchurl {
     url =
       "https://github.com/MonitorControl/${pname}/releases/download/v${version}/MonitorControl.${version}.dmg";
-    sha256 = "iaxM9j78Sq1EH5TCY240N+D5bG6quk2dZj8T7nt9ATo=";
+    sha256 = "Q96uK6wVe1D2uLvWL+pFR6LcmrU7cgmr2Y5tPvvTDgI=";
   };
 
-  nativeBuildInputs = [ undmg ];
```

Comment:
For sanity's sake, might good to explain why we're not using `undmg` to unwrap the .app.